test/Unit/DependencyManagement/DependencyManagementTests.cs

```diff
@@ -336,24 +336,27 @@ public void TestManagedDependencyRetryLogicMaxNumberOfTries()
 
                     var relevantLogs = _testLogger.FullLog.Where(
                         message => message.StartsWith("Error: Fail to install module")
-                                   || message.StartsWith("Trace: Installing FunctionApp dependent modules")).ToList();
+                                   || message.StartsWith("Trace: Installing FunctionApp dependent modules")
+                                   || message.StartsWith("Warning: Failed to install dependencies")).ToList();
 
                     // Here we will get four logs: one that says that we are installing the
```

Comment:
Please update the comment to reflect the code.

Reply:
Thank you, updated.